test/telemetry_metrics_cloudwatch_test.exs

```diff
@@ -172,12 +172,13 @@ defmodule TelemetryMetricsCloudwatchTest do
     end
 
     test "should keep values when given keep function" do
-      counter = Metrics.counter([:aname, :value], keep: &(Map.get(&1, :good) == 1))
+      counter = Metrics.counter([:aname, :value], keep: &(&1.good == 1 and &2.value <= 112))
```

Comment:
Hey @jaminthorns - I think my only other suggestion would be to ensure that keep/drop functions functions are tested with both an arity of 1 and 2 (so maybe keep the old tests and add two new ones that look like the changes in the PR).  Other than that, looks good to go!  Thanks for the help!

Reply:
Good point, added!